src/fetchAPI.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const myAPI = axios.create({
+    baseURL: "https://nc-news-app-kanti.herokuapp.com/api/",
+});
+
+export const fetchArticles = (sortBy) => {
+    if (!sortBy) {
+        return myAPI.get("/articles").then((res) => {
+            return res.data.articles;
+        });
+    } else {
+        return myAPI.get(`/articles?sort_by=${sortBy}`).then((res) => {
+            return res.data.articles;
+        });
+    }
+};
+
```

Comment:
Axios actually is very nice to deal with queries. For this one, we can put the second argument into mayAPI.get("/articles", {params:{sortBy: sort_by}}). Then we don't need the if statement anymore. Whichever query you're going to use, can simply put it into params. Axios will handle it for you

Reply:
Thanks for the tip Scarlett.